doc/api/errors.md

```diff
@@ -255,6 +255,13 @@ will affect any stack trace captured *after* the value has been changed.
 If set to a non-number value, or set to a negative number, stack traces will
 not capture any frames.
 
+#### error.code
```

Comment:
@seppevs ... yeah, I'll take care of this section in one of the other PRs. If you would, go ahead and pull this particular bit back out :-)